server/lib/s3backup/__init__.py

```diff
@@ -1,573 +0,0 @@
-"""
-This module provides convenience functions that interface to lower-level services, provided by the boto3 module.
-"""
```

Comment:
I wonder if there is value to archiving this work even though we are not using it now but if we decide to implement s3 backup in the future this could come in handy.

Reply:
We can always go back to the `v0.69.9` tag in the repo and pull these files from there...where/how would you archive it?

Reply:
Indeed.